man/docker-run.1.md

```diff
@@ -251,6 +252,11 @@ redirection on the host system.
 **-l**, **--label**=[]
    Set metadata on the container (e.g., --label com.example.key=value)
 
+**--isolation**="*default*"
+   Specify container's isolation level
+
+   It's quite a platform specific option which specifies isolation technology the container is going to use to isolate itself with others. For now, Linux support only 'default' (Linux Namespace), and Windows can support 'default' (Windows Server Containers) or 'hyperv' (Hyper-V Containers).
+
```

Comment:
All right, I'll change per your suggestion. Feel free to change any part that you think is inappropriate, because you are far more expert than me on this. :smile: 

Let's throw some information to end-users first.